ruthyn-example.c

```diff
@@ -0,0 +1,46 @@
+#include <stdio.h>
+
+/* ruthyn-example.c
+ *
+ * Tests out the the Unicode range between U+0400 and U+047F, which
+ * includes all the ruthyn-hd44780 characters. The string literals in
+ * this file are encoded in UTF-8 for easy development. When compiled
+ * correctly, this program actually emits the Ruthyn single byte
+ * encoding, not UTF-8.
+ *
+ * Compilation:
+ *	GCONV_PATH=../gconv gcc  -fexec-charset=ruthyn  ruthyn-example.c
+ *
+ * Required during compilation:
+ *	../gconv/gconv-modules				# Gconv catalog
+ *	../gconv/ruthyn-hd44780.so			# Gconv module
+ *
+ * Once compiled, the executable does _not_ require the gconv files.
+ *
+ * Note that this works only for literal strings in the code as the
+ * mapping happens at compilation time. Also, this method does not
+ * affect input.
```

Comment:
Frankly, did not get which input and how it might be affected.

Reply:
I will remove it as the sentence is confusing as is and the point is not important enough to spend a lot of words on.

For most embedded applications, there is no input. However, it is possible that a program may allow text input and would need the reverse mapping, from Ruthyn to Unicode.  Since bidirectional mapping is usually a given when using gconv modules, I thought it might be worthwhile to mention that the fexec-charset method only works for output.

Reply:
Yeah, the `gconv` solution is suitable for tasks of any scale. There are examples for complex use cases. The purpose of this project is exactly to extract that minimum suitable for MCU domain where unidirectional stateless conversion is used.
Will try to stipulate this in further updates.